src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -1820,34 +1925,34 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
 
         if (width && height) {
             videoCategory.stats.push({
-                label: globalize.translate('LabelVideoResolution'),
+                label: globalize.translate("LabelVideoResolution"),
                 value: width + 'x' + height
             });
         }
 
         if (mediaElement.getVideoPlaybackQuality) {
-            var playbackQuality = mediaElement.getVideoPlaybackQuality();
+            const playbackQuality = mediaElement.getVideoPlaybackQuality();
 
-            var droppedVideoFrames = playbackQuality.droppedVideoFrames || 0;
+            const droppedVideoFrames = playbackQuality.droppedVideoFrames || 0;
             videoCategory.stats.push({
-                label: globalize.translate('LabelDroppedFrames'),
+                label: globalize.translate("LabelDroppedFrames"),
                 value: droppedVideoFrames
             });
 
-            var corruptedVideoFrames = playbackQuality.corruptedVideoFrames || 0;
+            const corruptedVideoFrames = playbackQuality.corruptedVideoFrames || 0;
```

Comment:
```suggestion
                label: globalize.translate('LabelCorruptedFrames'),
```

Reply:
What's this all about?

Reply:
yeah ignore that I think i copied the wrong line

Reply:
that can be resolved